kyuubi-server/src/main/scala/org/apache/kyuubi/server/ui/JettyServer.scala

```diff
@@ -71,14 +74,20 @@ private[kyuubi] case class JettyServer(
   def getState: String = server.getState
 }
 
-object JettyServer {
+object JettyServer extends Logging{
 
-  def apply(name: String, host: String, port: Int, poolSize: Int): JettyServer = {
+  def apply(name: String, host: String, port: Int, kyuubiConf: KyuubiConf): JettyServer = {
+
+    // set Jetty Server pool size
```

Comment:
the comment usually should be a supplement to the code block which is composed of complex logic. And a good comment explains WHY should we do but not WHAT we are doing, because in most cases, the code is clear enough to answer the latter question

Reply:
I have removed unnecessary annotations.